docs/_adapters/nginx-sm.md

```diff
@@ -11,6 +11,23 @@ image: /assets/img/service-meshes/nginx-sm.svg
 permalink: service-meshes/adapters/nginx-sm
 ---
 
+{% assign sorted_tests_group = site.compatibility | group_by: "meshery-component" %}
+{% for group in sorted_tests_group %}
+      {% if group.name == "meshery-nginx-sm" %}
+        {% assign items = group.items | sort: "meshery-component-version" | reverse %}
+        {% for item in items %}
+          {% if item.meshery-component-version != "edge" %}
+            {% if item.overall-status == "passing" %}
+              {% assign adapter_version_dynamic = item.meshery-component-version %}
```

Comment:
Couldn't able to see version also here....

Reply:
![image](https://user-images.githubusercontent.com/114604338/209438891-292e21f8-7adf-42b2-bfdf-46e30adafdbe.png)
I can see the desired results on my local server.